.github/workflows/integration-tests.yml

```diff
@@ -0,0 +1,55 @@
+# End to end integration tests - GitHub Actions
+name: integration tests
+on: [push, pull_request]
+jobs:
+  integration-tests-push:
```

Comment:
rather than duplicating the whole workflow, could the conditional be on the `Prepare environment variable` step only?